csharp/src/Ice/MultiStreamSocket.cs

```diff
@@ -12,17 +12,22 @@
 
 namespace ZeroC.Ice
 {
-    /// <summary>The MultiStreamTransceiver abstract base class to implement multi-stream transports.</summary>
-    public abstract class MultiStreamTransceiver : IDisposable
+    /// <summary>A multi-stream socket enables transmitting raw binary data over multiple independent streams. The
```

Comment:
Would it be accurate to describe a MultiStreamSocket (and a SingleStreamSocket) as an object associated with an established connection? Like the local end of a connection?

Reply:
Sure.  So are you suggesting to fix the comment? Something like `A XXX-stream socket represents the local end of a network connection and enables transmitting raw binary data over ...` or do you have a better suggestion?

Reply:
It nows says: `A single-stream socket represents the local end of a network connection and enables transmitting raw binary data over a transport such as TCP, UDP or WebSocket.`